src/handlers/subjects/teacher.ts

```diff
@@ -0,0 +1,37 @@
+// GET
+import { CustomDynamoDB } from "../../dynamodb/database";
+import responseHelper from "../../helpers/responseHelper";
+import { validateToken } from "../../helpers/validations";
+
+const subjectsDB = new CustomDynamoDB('local-bootcamp-classroom-api-subjects', 'id')
+
+export const handler = async(event: any) => {
+    const tokenData = await validateToken(event.headers.Authorization)
+
+    if(!tokenData){
+        return responseHelper(403, 'Fail Authenticating');
+    }
+
+    if(!event.pathParameters || event.pathParameters.teacherEmail.trim() === '' || tokenData.userType !== "TEACHER"){
```

Comment:
For the validation of userType !== "TEACHER" make another if, and send another message